pa/pa.c

```diff
@@ -372,6 +386,29 @@ bool checkCommandExists(char *inputCommand)
     }
 }
 
+/*
+Print list of commands and actions
+*/
+void listCommands()
+{
+    // Get the file path
+    char directory[100];
+    getFilePath(directory, false);
+    FILE *commandFile;
+    commandFile = fopen(directory, "r");
+
+    char command[100];
+    char action[100];
+    while (fscanf(commandFile, fileFormatBoth, command, action) != EOF)
+    {
+        // Print command and action with colors
+        printf("%s%s: %s%s%s\n", RED, command, BLU, action, RESET);
```

Comment:
Remove the colors from the print statements. Also, it would be great if there were bullet points in front of the commands to indicate that it's a list.

```suggestion
        printf("* %s: %s\n", command, action);
```

Output would look like the following:
```
Jareds-MBP:~ jaredpetersen$ pa list commands
* what time is it: date +"%r"
* what day is it: date +"%A, %B %d %Y"
* shutdown pc: sudo shutdown -h now
* restart pc: sudo shutdown -r now
```